lib/wpc/include/msap.h

```diff
@@ -87,6 +88,12 @@ typedef struct __attribute__((__packed__))
     uint8_t param;
 } msap_scratchpad_write_req_pl_t;
 
+typedef struct __attribute__((__packed__))
+{
+    uint32_t start_add;
+    uint8_t number_of_bytes;
+} msap_image_read_req_pl_t;
```

Comment:
Shouldn't it be named msap_scratchpad_read_req_pl_t instead to be more aligned with write?

Reply:
Counterpart to `msap_image_read_req_pl_t` is `msap_image_block_req_pl_t`, not `msap_scratchpad_write_req_pl_t`, which is used to set the OTAP action and target instead.

Maybe `msap_scratchpad_write_req_pl_t` should be renamed to be more clear?